### PR TITLE
[To merge on 1 June] Marketplace API name change

### PR DESCRIPTION
**Pull request by natasha-moore-cko**
# Description of changes in PR

[//]: # 'Please add your description here'

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [x] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

File: nas_spec/paths/_platforms-files.yaml [hunk 1]
```diff
@@ -11,19 +11,19 @@ post:
       description: Sandbox server
   summary: Upload a file
   description: >-
-    Our <a href="https://checkout.com/docs/four/marketplaces" target="_blank">Marketplace</a> solution provides an easy way to upload identity documentation required for full due diligence. Please <strong>note</strong> that the sub-domain – https://files.checkout.com – is slightly different to Checkout.com's other endpoints. <br><br>Read the <a href="https://checkout.com/docs/four/marketplaces/onboarding/api/full/upload-a-file" target="_blank">documentation</a> for more information.
+    Our solution for platforms provides an easy way to upload identity documentation required for full due diligence. Please <strong>note</strong> that the sub-domain – https://files.checkout.com – is slightly different to Checkout.com's other endpoints. <br><br>Read the <a href="https://checkout.com/docs/four/platforms/onboarding/api/full/upload-a-file" target="_blank">documentation</a> for more information.
```
Comment (chrisi-webster-cko) [inline nas_spec/paths/_platforms-files.yaml:14]:
```suggestion
    Our Platforms solution provides an easy way to upload identity documentation required for full due diligence. Please <strong>note</strong> that the sub-domain – https://files.checkout.com – is slightly different to Checkout.com's other endpoints. <br><br>Read the <a href="https://checkout.com/docs/four/platforms/onboarding/api/full/upload-a-file" target="_blank">documentation</a> for more information.
```

Comment (chrisi-webster-cko) [reply nas_spec/paths/_platforms-files.yaml:14]: This is my bad for changing my mind and not changing the ticket.

File: nas_spec/paths/accounts@entities@{id}@payout-schedules.yaml [hunk 2]
```diff
@@ -13,10 +13,10 @@ get:
         type: string
   security:
     - OAuth:
-        - marketplace
+        - accounts
   summary: Retrieve a sub-entity's payout schedule
   description: >-
-    You can schedule when your sub-entities receive their funds using our <a href="https://checkout.com/docs/four/marketplaces" target="_blank">Marketplace</a> solution. Use this endpoint to retrieve information about a sub-entity's schedule.
+    You can schedule when your sub-entities receive their funds using our <a href="https://checkout.com/docs/four/platforms" target="_blank">solution for platforms</a>. Use this endpoint to retrieve information about a sub-entity's schedule.
```
Comment (chrisi-webster-cko) [inline nas_spec/paths/accounts@entities@{id}@payout-schedules.yaml:19]:
```suggestion
    You can schedule when your sub-entities receive their funds using our <a href="https://checkout.com/docs/four/platforms" target="_blank">Platforms solution</a>. Use this endpoint to retrieve information about a sub-entity's schedule.
```

File: nas_spec/paths/accounts@entities@{id}@payout-schedules.yaml [hunk 3]
```diff
@@ -40,13 +40,13 @@ get:
...
-    - Marketplace
+    - Platforms
   security:
     - OAuth:
-        - marketplace
+        - accounts
   summary: Update a sub-entity's payout schedule
   description: >-
-    You can schedule when your sub-entities receive their funds using our <a href="https://checkout.com/docs/four/marketplaces" target="_blank">Marketplace</a> solution. Use this endpoint to update a sub-entity's schedule.
+    You can schedule when your sub-entities receive their funds using our <a href="https://checkout.com/docs/four/platforms" target="_blank">solution for platforms</a>. Use this endpoint to update a sub-entity's schedule.
```
Comment (chrisi-webster-cko) [inline nas_spec/paths/accounts@entities@{id}@payout-schedules.yaml:49]:
```suggestion
    You can schedule when your sub-entities receive their funds using our <a href="https://checkout.com/docs/four/platforms" target="_blank">Platforms solution</a>. Use this endpoint to update a sub-entity's schedule.
```

**Review by kunal-patel-cko — APPROVED** [review]

**Review by chrisi-webster-cko — APPROVED** [review]

**Pull request merged**